### PR TITLE
fix: fix whitespace in docstrings

### PR DESCRIPTION
**Pull request by sebbacon**

### CODE REVIEW

File: cohortextractor/patients.py [hunk 1]
```diff
@@ -264,9 +259,9 @@ def most_recent_bmi(
 
     Args:
         on_or_before: date of interest as a string with the format `YYYY-MM-DD`. Filters results to measurements
-            on or before the given date. The default value is `None`.
+            on or before the given date.The default value is `None`.
```
Comment (wjchulme) [inline cohortextractor/patients.py:262]: did you mean to remove the space here?

File: cohortextractor/patients.py [hunk 2]
```diff
@@ -83,8 +82,7 @@ def age_as_of(
                 return_expectations={
                     "rate" : "universal",
                     "int" : {"distribution" : "population_ages"}
-                }
-            )
```
Comment (madwort) [inline cohortextractor/patients.py:87]: did you intend to remove this bracket?

File: cohortextractor/patients.py [hunk 3]
```diff
@@ -116,11 +114,8 @@ def date_of_birth(
 
             dob=patients.date_of_birth(
                 "YYYY-MM",
-                return_expectations={ 
-                    "date": {"earliest": "1950-01-01", "latest": "today"},
-                    "rate": "uniform",
-                }
-            )
```
Comment (madwort) [inline cohortextractor/patients.py:123]: did you intend to remove this bracket?

**Comment by inglesp** [comment]
@sebbacon can we close this?

**Pull request closed without merging**